### ISSUE TITLE
Question about DistributedDataParallel

### ISSUE THREAD

**Issue by geekboood** [mapillary/inplace_abn#103]
I only need to train my network with InplaceABN in single GPU and I have already read about `https://github.com/mapillary/inplace_abn/issues/84`. 
Now I have some problem about implementing `DistributedDataParallel` in my code. I wonder if it is necessary to use it when I only use `InPlaceABN` in my code.

**Comment by rotabulo** [comment]
@geekboood no that's not needed by `InPlaceABN`. It is required if you want to use the synchronized version.

**Comment by geekboood** [comment]
@rotabulo Thank you for the answer! By the way, I encounter a CUDA OOM problem when I change the learning rate, and I have never meet that when I using traditional BatchNorm. Do you have any idea about that? Is this InplaceABN need to consume more memory when I change the learning rate?

**Comment by rotabulo** [comment]
@geekboood no there is no relation at all, so you probably have other issues in the code.

**Comment by geekboood** [comment]
@rotabulo OK, thanks you for the answer. It turns out that although this InplaceABN could increase the batchsize and the performance somehow may benefit from that, in semantic segmentation scenario the performance of ResNeXt101 is worse than SE-ResNeXt50. Our dataset is relatively small and we must utilize pretrained models. Can I load the pretrained model for BN into a InplaceABN modified model? Will it save some memory when I disable the activation in InplaceABN and use it as a  normal BN?
